.gitignore

```diff
@@ -0,0 +1,21 @@
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
+
+# dependencies
+/node_modules
+
+# testing
+/coverage
+
+# production
+/build
+
+# misc
+.DS_Store
+.env.local
+.env.development.local
+.env.test.local
+.env.production.local
+
+npm-debug.log*
+yarn-debug.log*
+yarn-error.log*
```

Comment:
Add `package-lock.json` to the gitignore file

Reply:
Then delete the `package-lock.json` so that it is removed from the repo. And then you can create it again by saying `npm install`

Reply:
Deleted the package.json, pushed and then ran `npm install` to recreate the package-lock.json file